clients/web/src/app.js

```diff
@@ -182,7 +182,6 @@ girder.App = girder.View.extend({
         girder.router.navigate(route, {trigger: true});
 
         if (girder.currentUser) {
-            girder.eventStream.close();
```

Comment:
If I recall correctly, this `close` seemed necessary to release the request in cherrypy in a timely fashion.  We had originally configured cherrypy to have only 10 workers, and, since the event stream could hang on for minutes, it was possible to exhaust the available workers if this wasn't closed.  Are we now just relying on having enough workers?  Was there a problem with closing the stream?